### PR TITLE
Migrate to Pipenv and update Python version

### PR DESCRIPTION
**Pull request by EtienneM**
https://test-etienne-flask.osc-fr1.scalingo.io/

I don't have any more security vulnerability messages in my deployments logs:

```
 <-- Start deployment of test-etienne-flask -->
       Fetching source code
       Fetching deployment cache
-----> Clearing cached dependencies
-----> Installing python-3.8.2
-----> Installing pip
-----> Installing dependencies with Pipenv 2018.5.18…
       Installing dependencies from Pipfile.lock (6cbede)…
-----> Installing SQLite3
Sqlite3 successfully installed.
-----> Procfile declares types -> web
 Build complete, shipping your container...
 Waiting for your application to boot... 
 <-- https://test-etienne-flask.osc-fr1.scalingo.io -->
```

However the Dockerized version does not work locally yet... Should I spend more time trying to figure out how to do that or we don't care yet?

Related to https://github.com/Scalingo/one-stop-shop/issues/124

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
 
 The application is running at this url: https://sample-python-flask.scalingo.io/
+
+## Deploy in One Click
+
+[![Deploy to Scalingo](https://cdn.scalingo.com/deploy/button.svg)](https://my.osc-fr1.scalingo.com/deploy?source=https://github.com/Scalingo/sample-python-flask)
+
+## Running Locally
+
+The Docker solution does not fully work yet:
```
Comment (Soulou) [inline README.md]: Can you give more details about this, why doesn't it fully work yet?

Comment (EtienneM) [reply README.md]: It starts listening to the correct port, but in a virtualenv and I'm too bad in Python to understand how to get rid of this virtualenv to make this work...

**Comment by Soulou** [comment]
@EtienneM fixed, if my diff LGTY, please merge the rest LGTM

**Review by EtienneM — COMMENTED** [review]
Thanks!

**Pull request merged**